libs/ledger-live-common/src/exchange/platform/transfer/completeExchange.ts

```diff
@@ -86,11 +91,14 @@ const completeExchange = (
         if (unsubscribed) return;
 
         currentStep = "PROCESS_TRANSACTION";
-        await exchange.processTransaction(Buffer.from(binaryPayload, "hex"), estimatedFees);
+        const { payload, format }: { payload: Buffer; format: PayloadSignatureComputedFormat } =
+          isExchangeTypeNg(exchange.transactionType)
+            ? { payload: Buffer.from("." + binaryPayload), format: "jws" }
+            : { payload: Buffer.from(binaryPayload, "hex"), format: "raw" };
+        await exchange.processTransaction(payload, estimatedFees, format);
         if (unsubscribed) return;
 
-        const bufferSignature = Buffer.from(signature, "hex");
-        const goodSign = convertSignature(bufferSignature, exchangeType);
+        const goodSign = convertSignature(signature, exchange.transactionType);
```

Comment:
Add comment on why you reuse `exchange.transactionType` here instead of `exchangeType`: due to Mercuryo integration.
I'd prefer to find a better solution here :/

Reply:
transactionType is SELL, exchange.transactionType is SELLNG